ingest-asset-reconciler/src/test/scala/uk/gov/nationalarchives/ingestassetreconciler/LambdaTest.scala

```diff
@@ -232,6 +233,19 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach with TableDrivenPro
     argumentVerifier.verifyInvocationsAndArgumentsPassed(1, 0, 0, 0, 1, 0)
   }
 
+  "handler" should "return an error if more than one entity with the same asset name as SourceIdis" in {
```

Comment:
Hmm, maybe "return an error if more than one entity has the same asset name as its SourceId"